packages/core/test/integration/json.test.js

```diff
@@ -126,6 +126,31 @@ describe('model', () => {
       });
     }
 
+    if (current.dialect.name === 'mariadb') {
+      // MariaDB does not support native JSON type, it uses longtext instead
```

Comment:
To reduce the nesting, can you move this if-statement inside the test (and return if dialect !== mariadb)?
And move the comments inside the test as well

Reply:
I can do this but note that I've used the same syntax of the [preceding test](https://github.com/sequelize/sequelize/blob/b849ab47f59d0e08b766a029fee19ccd4fb80ea7/packages/core/test/integration/json.test.js#L112-L127)